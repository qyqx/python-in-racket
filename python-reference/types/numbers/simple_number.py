print(5);
print(3.3);
type(3);
