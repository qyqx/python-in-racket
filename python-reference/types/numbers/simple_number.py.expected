5
3.3
10
25.6
15
5
-5
-1.1
